### ISSUE TITLE
Category mapping doesn't accept correct scientific name capitalisation

### ISSUE THREAD

**Issue by geezacoleman** [Sydney-Informatics-Hub/Weed-AI#547]
In the dialogue box for category name mapping: `category name` is a `weed` of species `scientific name`, `scientific name` does not accept the capitalisation of Genus.

**Comment by jnothman** [comment]
Yes, currently our standard is to always have lowercase to ensure that there is consistency in multiple entries... Is that okay? Or should we use standard capitalisation? Perhaps we should transform the user's input to lowercase, or put a note in the form that lowrcase is expected.

**Comment by geezacoleman** [comment]
The capitalisation of genus on the explore page is probably something that should be fixed in the long term so it is technically correct, but in terms of priority it's quite far down. As for the form, it took me a bit of playing around to work out why it wasn't liking my scientific name, so accepting any form of capitalisation and transforming it would be ideal I think and probably fairly important thing to change so people have a smooth upload experience.

**Comment by jnothman** [comment]
So we should convert to lowercase for validation in frontend and backend, except for "UNSPECIFIED" which is expected uppercase.
